moveit_core/collision_distance_field/include/moveit/collision_distance_field/collision_env_distance_field.h

```diff
@@ -179,17 +179,22 @@ class CollisionEnvDistanceField : public CollisionEnv
 
   virtual double distanceRobot(const robot_state::RobotState& state, bool verbose = false) const
   {
+    (void)state;
```

Comment:
In the other files, you commented out the parameters in the function definition (e.g. `const robot_state::RobotState& /*state*/, bool verbose = false`), only in this one it's done differently. Is there a reason?

Reply:
Yes. I use three variantes:
1. Commenting
2. `(void)variable;`  (when commenting wasn't feasible due to default arguments defined)
3. Removing the variable (in cases where the type name already suggests the variable meaning, e.g. `NodeHandle`).

Reply:
I'm not sure of the exact rules, but it looks like you are able to comment with default arguments defined in other files, such as https://github.com/ros-planning/moveit/pull/1773/files?file-filters[]=.cpp&file-filters[]=.h&file-filters[]=.yml#diff-22ed9d5e496d7469a03e0b414299ebdeR919.

Reply:
Of course, when the default argument is in the header, I can comment (or omit) the variable name in the .cpp.

Reply:
I think `bool = false` or `bool /* verbose */ = false` for a default argument would also work. The latter seems clearer than the `(void)variable;` variant.